mopidy/core/library.py

```diff
@@ -189,14 +189,9 @@ def lookup(self, uri=None, uris=None):
         .. deprecated:: 1.0
             The ``uri`` argument. Use ``uris`` instead.
         """
-        none_set = uri is None and uris is None
-        both_set = uri is not None and uris is not None
-
-        if none_set or both_set:
+        if sum(o is not None for o in [uri, uris]) != 1:
             raise ValueError("One of 'uri' or 'uris' must be set")
```

Comment:
Switch to double quotes around the values?

Change to "Exactly one of ..." for consistency and clearity?